x/evm/keeper/grpc_query_test.go

```diff
@@ -981,3 +1125,164 @@ func (suite *KeeperTestSuite) TestQueryBaseFee() {
 	suite.enableFeemarket = false
 	suite.enableLondonHF = true
 }
+
+func (suite *KeeperTestSuite) TestEthCall() {
+	var (
+		req *types.EthCallRequest
+	)
+
+	suite.SetupTest()
```

Comment:
this should be placed inside the `suite.Run` below, the suite calls the setup when it's started so putting it here is not necessary